sql/src/main/java/io/crate/lucene/LuceneQueryBuilder.java

```diff
@@ -387,7 +386,8 @@ public Query apply(Function input, Context context) {
                 .put(LteOperator.NAME, LTE_QUERY)
                 .put(GteOperator.NAME, GTE_QUERY)
                 .put(GtOperator.NAME, GT_QUERY)
-                .put(LikeOperator.NAME, new LikeQuery())
+                .put(LikeOperators.OP_LIKE, new LikeQuery())
```

Comment:
Maybe we could pass in the ignore case parameter instead of extracting it inside the classes functions